packages/mobile/src/fees/saga.test.ts

```diff
@@ -1,12 +1,24 @@
+import BigNumber from 'bignumber.js'
 import { expectSaga } from 'redux-saga-test-plan'
-import { call, select } from 'redux-saga/effects'
-import { estimateFee, feeEstimated, FeeType } from 'src/fees/actions'
-import { watchEstimateFee } from 'src/fees/saga'
-import { getInvitationVerificationFee } from 'src/invite/saga'
-import { currentAccountSelector } from 'src/web3/selectors'
+import * as matchers from 'redux-saga-test-plan/matchers'
+import { call } from 'redux-saga/effects'
+import { getReclaimEscrowGas } from 'src/escrow/saga'
+import { feeEstimated, FeeType } from 'src/fees/actions'
+import { estimateFeeSaga } from 'src/fees/saga'
+import { getInvitationVerificationFee, getInviteTxGas } from 'src/invite/saga'
+import { getSendTxGas } from 'src/send/saga'
+import { getConnectedAccount } from 'src/web3/saga'
 import { mockAccount } from 'test/values'
 
-describe(watchEstimateFee, () => {
+const GAS_AMOUNT = 500000
+
+jest.mock('@celo/walletkit', () => ({
```

Comment:
for future reference since we will get rid of walletkit, but we should probably just create a mock file for contractkit. or would be great to ship the package with a mocks file itself even - do people do that ever?

Reply:
Agreed, a shared mock would be better. Might not fix in this Pr tho